cnv/cnv_install/installing-container-native-virtualization.adoc

```diff
@@ -0,0 +1,30 @@
+[id="installing-container-native-virtualization"]
+= Installing {ProductName}
+include::modules/cnv-document-attributes.adoc[]
+:context: installing-container-native-virtualization
+toc::[]
+
+Before deploying {ProductName}, use the CLI to prepare the required namespace,
+`OperatorGroup`, and `CatalogSource`.
+
+Then, you can install {ProductName} by using the {product-title} 4.1
+xref:../../web-console/web-console.adoc#web-console-overview_web-console[web console].
+
+.Prerequisites
+
+* {product-title} 4.1
+// link to requirements assembly for more details about cluster
+* User with `cluster-admin` privileges
+
```

Comment:
Do you have a concept module that provides details about the installation process? I think this assembly might benefit from one, even if it's not much more than "You need to install OCP before you install CNV. CNV runs as an Operator in the `<namespace>` namespace for \<reasons>\. Here's anything else you need to consider before you install."